components/brave_ads/core/internal/ad_units/ad_handler.h

```diff
@@ -9,6 +9,8 @@
 #include <cstdint>
 #include <string>
 
+#include "base/scoped_multi_source_observation.h"
```

Comment:
scoped_multi_source_observation.h is not used

Reply:
Fixed, thanks